### ISSUE TITLE
hasCallbackQuery call BotCommand

### ISSUE THREAD

**Issue by igson** [rubenlagus/TelegramBots#702]
It is possible to hasCallbackQuery call a BotComand?

**Comment by Chase22** [comment]
I don't quite understand the question.

What do you want to do?

**Comment by igson** [comment]
`@Override
	public void processNonCommandUpdate(Update update)  {
		
	 if (update.hasMessage()) {
 
         } else if (update.hasCallbackQuery()) {
        
        	log.info("MENSAGEM DE HasCallbackQuery....");
 
           	String chamada = update.getCallbackQuery().getData();
        	
            if ( chamada.equals("lista") ) {
                **I want to call a BotCommand here...**
            }
        	
	}`

**Comment by Chase22** [comment]
What exactly do you mean by calling a bot command?
Sending a message etc?

**Comment by igson** [comment]
> What exactly do you mean by calling a bot command?
> Sending a message etc?

Yes. It is possible?

**Comment by igson** [comment]
Yes. It is possible?

**Comment by Chase22** [comment]
Yes it's definetly possible. You can still access all normal bot methods, like execute()

**Comment by igson** [comment]
> Yes it's definetly possible. You can still access all normal bot methods, like execute()

_**Have an example of how to do this?**_

**Comment by Chase22** [comment]
@igson 
```
SendMessage message = new SendMessage()
                .setChatId(theChatId)
                .setText(theMessageToSend);
        try {
            execute(message);
        } catch (TelegramApiException e) {
            e.printStackTrace();
        }
```

from: [Getting Started](https://github.com/rubenlagus/TelegramBots/wiki/Getting-Started#build-our-first-bot)
